### ISSUE TITLE
Any option to change the decimal place from point to comma?

### ISSUE THREAD

**Issue by darkdroider** [Solidet-com/steam-currency-converter#3]
Any option to change the decimal place from point to comma?

R$ 8.88 to R$ 8,88  ?

Thanks

**Comment by Trsnaqe** [comment]
Hello, Thank you so much for the feedback! 

We are putting dots and commas according to currency's own formatting data we have prepared before the 1.0 update. While searching, we retrieved BRL data from [this website](https://documentation.infiniteblue.com/platform/Currency%20formats.htm) then converted to data below:

```json
{
  "id": 7,
  "abbr": "BRL",
  "symbol": "R$",
  "hint": "Brazilian Reals",
  "multiplier": 100,
  "unit": 1,
  "format": {
    "places": 2,
    "hidePlacesWhenZero": false,
    "symbolFormat": "R$ ",
    "thousand": ".",
    "decimal": ",",
    "right": false
  }
}
```

Upon reevaluation, we found information that supports your claim on [this website](https://brazil-help.com/g_currency.htm) and [this website](https://www.brazilcultureandtravel.com/brazil-currency.html). To confirm it for sure I have checked [Banco Central Do Brasil ](https://www.bcb.gov.br/) and thousand seperator is indeed "," so we will have it updated by the next update. The first website I have come across must be wrong. 

Thanks for your feedback again!

![Brazil Central Bank Currency Formattin Picture](https://i.imgur.com/Up15Hui.png)

**Comment by Trsnaqe** [comment]
Thank you for the issue!

Even though I have marked this as currency formatting specific I have actually noticed that we have not been formatting the converted price according to our currency formatting, I will be deploying an update later today with the new version. 

Thanks for the heads up!

**Comment by darkdroider** [comment]
thanks bro...
You are a hero
    Em sábado, 16 de março de 2024 às 10:24:38 BRT, Mahmut Sacit Meydanal ***@***.***> escreveu:  
 
 


Closed #3 as completed via 00948b2.
